examples/device/net_lwip_webserver/src/main.c

```diff
@@ -178,21 +178,12 @@ bool tud_network_recv_cb(const uint8_t *src, uint16_t size)
 uint16_t tud_network_xmit_cb(uint8_t *dst, void *ref, uint16_t arg)
 {
   struct pbuf *p = (struct pbuf *)ref;
-  struct pbuf *q;
-  uint16_t len = 0;
 
   (void)arg; /* unused for this example */
 
-  /* traverse the "pbuf chain"; see ./lwip/src/core/pbuf.c for more info */
-  for(q = p; q != NULL; q = q->next)
-  {
-    memcpy(dst, (char *)q->payload, q->len);
-    dst += q->len;
-    len += q->len;
-    if (q->len == q->tot_len) break;
-  }
+  pbuf_copy_partial(p, dst, p->tot_len, 0);
 
-  return len;
+  return p->tot_len;
```

Comment:
Since `pbuf_copy_partial` returns 'the number of bytes copied, or 0 on failure' it might be more logical to return this value instead of always `p->tot_len`.

Reply:
That makes sense; I have added a commit that does just that.  Thank you for the review and the suggestion!